README.md

```diff
@@ -20,59 +19,53 @@ var initTracer = require('jaeger-client').initTracer;
 
 // See schema https://github.com/jaegertracing/jaeger-client-node/blob/master/src/configuration.js#L37
 var config = {
-  'serviceName': 'my-awesome-service'
+  serviceName: 'my-awesome-service',
 };
 var options = {
-  'tags': {
-    'my-awesome-service.version': '1.1.2'
+  tags: {
```

Comment:
@tiffon is this valid ES5 javascript? I thought quotes are required there.

Reply:
Yes, it's valid. And, the trailing comma on the last property in object literals (`logger: logger,`) [is also fine](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#Objects).